src/com/facebook/buck/python/AbstractPythonVersion.java

```diff
@@ -25,11 +25,22 @@
 abstract class AbstractPythonVersion {
 
   @Value.Parameter
-  public abstract String getVersionString();
+  public abstract String getInterpreterName();
+
+  @Value.Parameter
+  public abstract String getVersionString();  // X.Y.Z
+
+  public String getPexCompatibilityVersion() {
+    StringBuilder sb = new StringBuilder(20);
+    sb.append(getInterpreterName());
+    sb.append(" ");
+    sb.append(getVersionString().replace('.', ' '));
+    return sb.toString();
+  }
```

Comment:
It'd be nice if we didn't have more PEX-specific stuff in the core python support.  Can we just have `PythonVersion` support the interpreter-name and version (as you've added above) and have the make_pex.py script do the necessary PEX-specific formatting?